Cargo.toml

```diff
@@ -1,37 +1,54 @@
 [workspace]
 resolver = "2"
 members = [
-  "fendermint/abci",
-  "fendermint/app",
-  "fendermint/app/options",
-  "fendermint/crypto",
-  "fendermint/app/settings",
-  "fendermint/eth/*",
-  "fendermint/rocksdb",
-  "fendermint/rpc",
-  "fendermint/storage",
-  "fendermint/testing",
-  "fendermint/testing/*-test",
-  "fendermint/vm/*",
+  # contracts
+  "contracts/binding",
+  # fvm-utils,
+  "fvm-utils/primitives",
+  "fvm-utils/runtime",
```

Comment:
As part of the monorepo I would like to deprecate `fvm-utils`. Now that we don't need the `ffi` and we use solidity contracts the repo is no longer needed and we can move the primitives to the monorepo directly. 
```suggestion
```

Reply:
Thanks, I saw that primitives have been moved, just wanted to defer that after it's all done.